uint16 chassis_volt
uint16 chassis_current
float32 chassis_power
uint16 chassis_power_buffer
uint16 shooter_heat_cooling_limit
uint16 shooter_heat
uint16 robot_hp
float32 bullet_speed
uint16 hurt_armor_id
uint16 hurt_type



time stamp
